### PR TITLE
Change type of offset in utility_meter

### PR DESCRIPTION
**Pull request by dgomes**
**Description:**

Updates the type of **offset** from integer to **time** and provides an example of how to define a Time period (copy&paste from sensor.darksky)

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20926

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/utility_meter.markdown [hunk 1]
```diff
@@ -42,17 +42,27 @@ cycle:
   required: true
   type: string
 offset:
-  description: Cycle reset occur at the beginning of the period (0 minutes, 0h00 hours, Monday, day 1, January). This option enables the offsetting of these beginnings.
+  description: Cycle reset occur at the beginning of the period (0 minutes, 0h00 hours, Monday, day 1, January). This option enables the offsetting of these beginnings. Supported formats: `offset: 'HH:MM:SS'`, `offset: 'HH:MM'` and Time period dictionary (see example below)."
```
Comment (klaasnicolaas) [inline source/_components/utility_meter.markdown]:
```suggestion
  description: "Cycle reset occur at the beginning of the period (0 minutes, 0h00 hours, Monday, day 1, January). This option enables the offsetting of these beginnings. Supported formats: `offset: 'HH:MM:SS'`, `offset: 'HH:MM'` and Time period dictionary (see example below)."
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by dgomes** [comment]
@klaasnicolaas tks!

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by cgarwood** [comment]
@dgomes looks like there's a merge conflict with the file now, otherwise I think the backend and this PR are ready to merge. Guessing it will land in 0.90?

**Comment by dgomes** [comment]
@cgarwood thanks for the heads up

Beta was already cut, don't think it qualifies as a bug fix to be fit in 0.89

**Review by cgarwood — APPROVED** [review]

**Pull request merged**